### ISSUE TITLE
Support opening multiple terminal apps

### ISSUE THREAD

**Issue by agileapricot** [wbond/sublime_terminal#16]
I don't know if there is a way to do this already (i couldn't see one), but for half my projects I need to the windows power shell open and for the other half I need Cygwin's mintty. Can it be possible to create two different commands for each?

**Comment by twolfson** [comment]
I could foresee us adding support by allowing people to pass in `terminal` to `run_terminal` and use a fallback for `TerminalSelector.get()`. In your usage, it would be a new keybinding/command to invoke `OpenTerminal` with the different `terminal`:

https://github.com/wbond/sublime_terminal/blob/1.4.0/Terminal.py#L115

```json
{ "keys": ["ctrl+shift+t"], "command": "open_terminal", "args": {"terminal": "mintty"} },
```

@wbond Can we add the default set of GitHub labels? I want to mark this with a "help wanted" and "enhancement" label.

**Comment by twolfson** [comment]
This has been resolved by @PinataMostGrim in #206 and released in `1.20.0`
